### ISSUE TITLE
Use docker for selenium Chrome Driver 

### ISSUE THREAD

**Issue by ngaruko** [medic/cht-core#7282]
**Describe the issue**
At times, when Google chrome version gets updated, our e2e tests fail to 'create session, like in [this build ](https://github.com/medic/cht-core/pull/7278/checks?check_run_id=3458535338), with errors:

```
[05:31:28] E/runner - Unable to start a WebDriver session.
(node:3092) UnhandledPromiseRejectionWarning: SessionNotCreatedError: session not created: This version of ChromeDriver only supports Chrome version 90
Current browser version is 92.0.4515.131 with binary path /usr/bin/google-chrome
  (Driver info: chromedriver=90.0.4430.24 (4c6d850f087da467d926e8eddb76550aed655991-refs/branch-heads/4430@{#429}),platform=Linux 5.4.0-1055-azure x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 386 milliseconds
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:25:53'
System info: host: 'fv-az54-435', ip: '10.1.0.26', os.name: 'Linux', os.arch: 'amd64', os.version: '5.4.0-1055-azure', java.version: '1.8.0_292'
Driver info: driver.version: unknown
```

This error comes when the version of the ChromeDriver and the version of the local browser don't match. We seem to have managed it somehow by using a [script](https://github.com/medic/cht-core/blob/master/scripts/e2e/start_webdriver.sh)  that checks first for the chrome version installed on the machine and grabs the version from there. While this solution seems to work so far, we there are 2 issues I can think of here:
1 - the Github machine won't always give us the desired chrome, so we are stuck with whatever version is on the machine.
2 - mismatch of versions: if chrome is updated before chrome driver (say chrome is v94 and chrome driver is still v93), in this case we the script won't be able to find a chrome driver version not published yet.

**Describe the improvement you'd like**
So to solve this problem we can use Docker images that come with specific matching (and tested) versions of ChromeDriver and Chrome browser installed. 
`docker run -d -p 4444:4444 --shm-size=2g selenium/standalone-chrome:<tag>`

Another advantage that we get with Docker is that we can have multiple Chrome versions set up which we can use to execute our tests, once we have figured out how to run tests with multicapabilities, in parallel mode.

At the moment, the easy and dirty way is to use an existing selenium/standalone-chrome (https://hub.docker.com/r/selenium/standalone-chrome/tags?page=1&ordering=last_updated) image. In the future, we could decide to build our own image with a certain version of chrome, but at the moment, it seems selenium keeps things up to date in docker hub(https://hub.docker.com/u/selenium), so we could use the latest or, if needed pick up an older tag. Or even use multiple Chrome versions set up, once we have figured out how to run tests with `multicapabilities`, on different ports.

**Describe alternatives you've considered**
script (https://github.com/medic/cht-core/blob/master/scripts/e2e/start_webdriver.sh) seems to work so far.

**Comment by garethbowen** [comment]
@ngaruko The associated PR was closed as as "no longer needed". Does that mean we can close this issue now?

**Comment by ngaruko** [comment]
@garethbowen Yes. These were protractor tests. WebdriverIO manages chrome differently - and we don't have this issue any longer. Closing this.
